### ISSUE TITLE
AUTHCODE IS ALWAYS EMPTY

### ISSUE THREAD

**Issue by mckoupel** [googlesamples/google-signin-unity#71]
I can't get authcode after request. In my configuration i set RequestAuthCode = true...nothing 
I've even created new google account and sign in  to application...and on first time authorisation I don't have this string, but have other data (f. e. DisplayName, IDtoken or Email even RequestEmail = false in configuration). Can I do something with this... Maybe exchange IdToken to AuthCode using WebRequest or something else?

**Comment by novavision** [comment]
Same issue.. impossible to proceed

**Comment by novavision** [comment]
@mckoupel I got the same issue, but seems it's usual google service behaviour. In my case I use google login for Firebase and just realised that after 1st google account login I can use just a IdToken without AuthCode just passing an empty parameter to 
`Credential credential = GoogleAuthProvider.GetCredential(idToken, authCode);`

Hope that will help

**Comment by shadesbelow** [comment]
This is a hack but if you call logout/signout before login, it seems to always give an AuthCode.

**Comment by thsbrown** [comment]
Yeah i've experience this too. Actually this problem for me only seems to happen on iOS.

**Comment by AQ1991** [comment]
Yeah tried it does not work
